docs/sections/getting_started/basics.md

```diff
@@ -498,32 +489,26 @@ As long as the first part of the UUID is unique within your database, AiiDA will
 To execute the `CalcJob`, you need to feed it (together with the inputs) to the `run` function provided by the AiiDA engine:
 
 ```{code-block} ipython
-
 In [4]: from aiida.engine import run
    ...: run(ArithmeticAdd, code=code, x=x, y=y)
-
 ```
 
 Wait for the process to complete, as it may take a couple of seconds.
 Once it is done, it will return a dictionary with the output nodes:
 
 ```{code-block} ipython
-
 Out[4]:
 {'sum': <Int: uuid: 7d5d781e-8f17-498a-b3d5-dbbd3488b935 (pk: 8) value: 11>,
 'remote_folder': <RemoteData: uuid: 888d654a-65fb-4da0-b3bc-d63f0374f274 (pk: 9)>,
 'retrieved': <FolderData: uuid: 4733aa78-2e2f-4aeb-8e09-c5cfb58553db (pk: 10s)>}
-
 ```
 
 Besides the sum of the two `Int` nodes, the calculation function also returns two other outputs: one of type `RemoteData` and one of type `FolderData`.
 See the {ref}`topics section on calculation jobs <topics:calculations:usage:calcfunctions>` for more details.
 Now, exit the IPython shell and once more check for *all* processes:
 
 ```{code-block} console
-
 $ verdi process list -a
```

Comment:
Now that I notice I think in the calculations section we used `--all`. Maybe we should check for consistency in this?

Reply:
I would indeed switch to `--all` here, and then maybe use `-a` at some other point and explain that it's a shorthand.

Reply:
Hmm, now that I've changed it, I'm not so sure anymore. I've added `-all` in the sentence before it ([...] simply add the `-a/--all` option), but I think we should just have them use the shorthands right away, else they might adopt a bad habit from the start.